doc/adr/003-dev-test-release-process.md

```diff
@@ -0,0 +1,91 @@
+# A short descriptive title - what is the outcome of this adr?
+
+-   Status: Proposed
+-   Deciders: Team
+-   Date: 15.11.2024
+
+## Result
+
+Which alternative was chosen. Do not include any reasoning here.
+
+## Problem context
+
+We want to improve several aspects of SDLC for team Apps.
+
+Development and testing:
+* Low confidence in the state of `main` leads to lower release cadence
+  * PRs in frontend aren't blocked from merging if e.g. some tests fail, partially due to flake tests (Cypress)
+  * PRs are sometimes merged before they are manually tested
+    * Partially due to the existing process
+    * Partially due to missing infrastructure for testing unmerged code in e.g. tt02
+* Frontend and backend development work is disjoint
+  * Leads to less cohesive code, worse APIs
+  * Slower development time due to handover
+  * Worse testing (parts of the stack are tested in isolation, as opposed to the whole of a feature/fix)
+  * Team members have expressed desire to work full-stack
+  * Repo separation inhibits tooling and automation across the full stack (e.g. integration tests across frontend and backend within a PR/feature branch)
+* Code review process is sometimes slow
+  * Time spent in review is not tracked - both with respect to the PR/work item itself and time spent by devs reviewing
+  * No one is "allocated" to reviews
+  * Different devs have different habits
+* No dedicated tester
+  * Developers test their own work (or reviewers in the same "task force")
+  * Dedicated testers bring a better process and competence
```

Comment:
Minor point, this isn't really covered in the proposed solutions I suppose? Should we recommend getting a dedicated person to do this or should we modify our testing process somehow?